### DISCUSSION TITLE
Trascr_014: indicazioni su genitori biologici

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1038] [Supporto servizi cooperativi] [answered]
In 
_Trascr_014 - Trascrizione della sentenza di adozione di minore in casi particolari emessa dal Tribunale dei minori_
non sono presenti la sezioni
"Madre biologica"
"Padre biologico"
che dovrebbero esserci dal momento che in questo procedimento specifico ha certamente senso documentare la distinzione tra genitori adottanti e genitori biologici. 
Le sezioni 
"Madre biologica"
"Padre biologico"
si hanno invece nei caso d'uso:
_Trascr_013 - Trascrizione di nascita: Trascrizione della sentenza di adozione di minore emessa dal Tribunale dei minori
Trascr_016 - Trascrizione di nascita: Trascrizione della sentenza di adozione di maggiorenne emessa dal Tribunale_
dove non ci risulta debba mai essere documentata la distinzione tra genitori adottanti e genitori biologici. 

Potreste verificare e darci notizie/aggiornamenti in merito?

**Comment by acolosimosogei** [comment]
Salve, stiamo lavorando alla discussion #1038, le daremo un riscontro quanto prima.
Grazie.

**Comment by asqueglia** [comment, marked as answer]
Buongiorno, al momento il sistema  non prevede l'acquisizione dei genitori biologici nelle adozioni di casi particolari, in quanto questi sono presenti nell'atto di nascita. Il sistema utilizza tali informazioni presenti nell'atto di nascita, per emettere la certificazione laddove l'adottante sia solo uno dei due genitori. Faremo un ulteriore passaggio con ANUSCA per capire se in questo modo copriamo tutte le casistiche.
Comunichiamo che c'è una issue di interesse da poter seguire #1149

**Comment by antonioluberto** [comment]
Qualcosa non è chiaro: dal momento che le sezioni dati riferite a “padre biologico” / “madre biologica” sono presenti nella modellazione relativa ai casi d’uso 
Trascr_013 - Trascrizione di nascita: Trascrizione della sentenza di adozione di minore emessa dal Tribunale dei minori
Trascr_016 - Trascrizione di nascita: Trascrizione della sentenza di adozione di maggiorenne emessa dal Tribunale
parrebbe che nozione in questo senso sia prevista su ANSC.
Tuttavia: il riferimento alle informazioni “padre biologico” / “madre biologica”:
-	lo troviamo per Trascr_013 e Trascr_016, dove normativamente non ci risulta debbano essere documentate
-	non lo troviamo per Trascr_016 dove normativamente ci risulta debbano essere documentate
A questo punto:
a)	dobbiamo supporre verranno rimosse anche per Trascr_013 e Trascr_016
oppure
b)	ci date indicazioni su quali siano le situazioni in corrispondenza delle quali in Trascr_013 e Trascr_016 vi aspettate possano esserci “padre biologico” / “madre biologica”?

**Comment by asqueglia** [comment]
Buongiorno come concordato in call è stata riaperta la issue https://github.com/italia/ansc/issues/1149 in modo da prendere in carico l'aggiornamento del caso d'uso. Si prega di far riferimento alla issue
